packages/ui/src/lib/modal/Modal.svelte

```diff
@@ -4,15 +4,17 @@ import { createEventDispatcher, onDestroy } from 'svelte';
 import { tabWithinParent } from '../utils/dialog-utils';
 
 const dispatch = createEventDispatcher();
-const close = (): boolean => dispatch('close');
 
 let modal: HTMLDivElement;
 export let name = '';
 export let top: boolean = false;
+export let onclose: () => void = () => {
```

Comment:
onClose ?

Reply:
it seems svelte is using onclick, onclose (vs on:click) (not onClose)

Reply:
Okey weird camel rules but okey